scss/_base_code.scss

```diff
@@ -20,10 +21,10 @@
   }
 
   pre {
-    background: $color-x-dark;
-    border: 0;
-    border-radius: .5rem;
-    color: $color-mid-dark;
+    background-color: $color-light;
+    border: 1px solid $color-mid-dark;
+    border-radius: 2px;
```

Comment:
It's frustrating to see declarations but no explanations. If all other units in a pattern are relative, why would this unit be static?